### DISCUSSION TITLE
no exported member 'RouteLookup'

### DISCUSSION THREAD

**Discussion by fidelio314** [feathersjs/feathers#3351] [Q&A]
### Steps to reproduce
When executing on a new typescript project (v5) the 
`npm run compile`

i have this error when compiling:
The scenario is: 

```
npm create feathers@latest feather-app
```

then 
selected: express as transport and REST (no real time) + SQL db adapter.
Then 
```
npm install
```
then 

```
> npm run compile

> feather-app@0.0.0 compile
> shx rm -rf lib/ && tsc

node_modules/@feathersjs/express/lib/declarations.d.ts:4:119 - error TS2305: Module '"@feathersjs/feathers"' has no exported member 'RouteLookup'.

4 import { Application as FeathersApplication, Params as FeathersParams, HookContext, ServiceMethods, ServiceInterface, RouteLookup } from '@feathersjs/feathers';
                                                                                                                        ~~~~~~~~~~~


Found 1 error in node_modules/@feathersjs/express/lib/declarations.d.ts:4
```

### Expected behavior

The typescript compilation should work

### Actual behavior

Constantly compiling breaks. 

### System configuration

**Module versions** feathesjs versions: 

```
"@feathersjs/adapter-commons": "^5.0.11",
"@feathersjs/authentication": "^5.0.11",
"@feathersjs/authentication-client": "^5.0.11",
"@feathersjs/configuration": "^5.0.11",
"@feathersjs/errors": "^5.0.11",
"@feathersjs/express": "^5.0.11",
"@feathersjs/feathers": "^5.0.11",
"@feathersjs/knex": "^5.0.11",
"@feathersjs/schema": "^5.0.11",
"@feathersjs/transport-commons": "^5.0.11",
"@feathersjs/typebox": "^5.0.11",
```

**NodeJS version**: 18.18.2

**Operating System**: ubuntu 23.10

**Comment by fidelio314** [comment]
it seems that the decalration of RouteLookup was added on this file packages/express/src/declarations.ts on this commit

https://github.com/feathersjs/feathers/commit/26aa937c114fb8596dfefc599b1f53cead69c159#diff-8a215d34dc2acdeff0871a1946ad1c914ebf5c039cfdb080c994c3d5652d6ac9
"
thought i dont understand why as the package feathersjs/feathers does not export it

**Comment by daffl** [comment]
I am not able to reproduce this issue (but ran into #3319 which will be fixed in the next release) when generating an app with the following options:

<img width="1403" alt="Screenshot 2023-11-27 at 10 57 49" src="https://github.com/feathersjs/feathers/assets/338316/a07242f1-4fe3-40fa-9b80-0b7826817e48">

You can however try to `import '@feathersjs/transport-commons'` in your `app.ts` file which may fix this as well.

**Reply by diranged** [reply]
I wanted to follow up and say that this is still an issue with Feathers JS /Express 5.0.31... but, adding `import '@feathersjs/transport-commons';` to the code does resolve the issue. Is there a bug though to make this simpler? If it helps, in our case we're building an ESM module with this code.
